doc/classes/SkeletonModification2DTwoBoneIK.xml

```diff
@@ -0,0 +1,94 @@
+		TwoBoneIK is great for arms, legs, and really any joints that can be represented by just two bones that bend to reach a target. This solver is more lightweight than [SkeletonModification2DFABRIK], but gives similar, natural looking results.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="get_joint_one_bone2d_node" qualifiers="const">
+			<return type="NodePath">
+			</return>
+			<description>
+				Returns the [Bone2D] node that is being used as the first bone in the TwoBoneIK modification.
+			</description>
+		</method>
+		<method name="get_joint_one_bone_idx" qualifiers="const">
+			<return type="int">
+			</return>
+			<description>
+				Returns the index of the [Bone2D] node that is being used as the first bone in the TwoBoneIK modification.
+			</description>
+		</method>
+		<method name="get_joint_two_bone2d_node" qualifiers="const">
+			<return type="NodePath">
+			</return>
+			<description>
+				Returns the [Bone2D] node that is being used as the second bone in the TwoBoneIK modification.
+			</description>
+		</method>
+		<method name="get_joint_two_bone_idx" qualifiers="const">
+			<return type="int">
+			</return>
+			<description>
+				Returns the index of the [Bone2D] node that is being used as the second bone in the TwoBoneIK modification.
+			</description>
+		</method>
+		<method name="set_joint_one_bone2d_node">
+			<return type="void">
+			</return>
+			<argument index="0" name="bone2d_node" type="NodePath">
+			</argument>
+			<description>
+				Sets the [Bone2D] node that is being used as the first bone in the TwoBoneIK modification.
+			</description>
+		</method>
+		<method name="set_joint_one_bone_idx">
+			<return type="void">
+			</return>
+			<argument index="0" name="bone_idx" type="int">
+			</argument>
+			<description>
+				Sets the index of the [Bone2D] node that is being used as the first bone in the TwoBoneIK modification.
+			</description>
+		</method>
+		<method name="set_joint_two_bone2d_node">
+			<return type="void">
+			</return>
+			<argument index="0" name="bone2d_node" type="NodePath">
+			</argument>
+			<description>
+				Sets the [Bone2D] node that is being used as the second bone in the TwoBoneIK modification.
+			</description>
+		</method>
+		<method name="set_joint_two_bone_idx">
+			<return type="void">
+			</return>
+			<argument index="0" name="bone_idx" type="int">
+			</argument>
+			<description>
+				Sets the index of the [Bone2D] node that is being used as the second bone in the TwoBoneIK modification.
+			</description>
+		</method>
+	</methods>
+	<members>
+		<member name="flip_bend_direction" type="bool" setter="set_flip_bend_direction" getter="get_flip_bend_direction" default="false">
+			When [code]true[/code], the bones in the modification will blend outward as opposed to inwards when contracting. When [code]false[/code], the bones will bend inwards when contracting.
```

Comment:
```suggestion
			If [code]true[/code], the bones in the modification will blend outward as opposed to inwards when contracting. If [code]false[/code], the bones will bend inwards when contracting.
```

(It's not wrong per se with "When" but we standardize on using "If [code]true[/code], ..." for describing boolean properties.)